Makefile

```diff
@@ -176,6 +179,8 @@ black: compose-build
 .PHONY: clean
 clean:
 	@echo "Cleaning project temporary files and installed dependencies..."
+	@docker rm -f $(DOCKER_CONTAINER_IDS) || echo "no containers to remove"
+	@docker volume rm $(DOCKER_VOLUME_IDS) || echo "no volumes to remove"
```

Comment:
Should this stop running containers first? 

I tested with the typical containers up and running and got errors about not being able to kill running containers, and saw both "no containers to remove" and "no volumes to remove":

```
Cleaning project temporary files and installed dependencies...
80400ba2b0d9
e41f997b75bf
7e7b024904de
2c7e8cb8e210
7f00b3ccc929
1d02b229a229
8301e1daffeb
5bf97b689c90
8413e7197ee1
cd069ad12247
710087230ea0
dc568dd02cc5
4b71bf30e55d
b2d2851422da
0f91549ab17b
75d8179239fa
be3175b525a1
667e302dce83
e79c3961c40c
e08556d17614
f043c0b58a28
6e671b28209b
04b58259fc3c
2bf3b38b5fdb
2a92ecb0fc93
60789245d4f8
2166d05746ce
de37b604f944
f94b511abb8f
36aed1a574fc
6a13382a5212
c38c8a67cfe8
474f6cace71b
c3d54e21d35a
d477b16c8635
0393ca54ae5e
6e709a5dad4c
df92d86d5ef7
27da19761371
4d1d6bc21604
a04ccc9581ed
cdad7774cacd
f5c3d8cdcee7
9ee0b11a4892
ecf01a2f64c7
c37c53fc9bbe
092f95509089
fcde32e14f27
55ed17628949
cac481be18c1
1f944de09b4b
f8a2212eba34
3452444855b7
6c55a5065999
1a462eb81b55
ecb78a7bb0e0
f5c5cb261935
a7623c775448
9343aa85a3b0
eef1e8fabef3
ef51bdfd7d17
1e9c8ff1fd89
Error response from daemon: Could not kill running container 9726f1077a7396f547e7c219c0229a2a7a9d4a308359a745a64eff1dd6c8a627, cannot remove - container 9726f1077a73 PID 92233 is zombie and can not be killed. Use the --init option when creating containers to run an init inside the container that forwards signals and reaps processes
Error response from daemon: Could not kill running container 4dd064c3224874d728ab621a3b82970f6cfe35303cbe68ba60c063b68cfd2fff, cannot remove - tried to kill container, but did not receive an exit event
Error response from daemon: Could not kill running container a02f61cd12103fb5d1a4c015c5bf7e35074d74f1a5eeca54e0cdca741b8565b1, cannot remove - container a02f61cd1210 PID 90315 is zombie and can not be killed. Use the --init option when creating containers to run an init inside the container that forwards signals and reaps processes
Error response from daemon: Could not kill running container f0972005382a684aa39a62e179a21b34c01223b1798d992d60513e8eaec923e7, cannot remove - tried to kill container, but did not receive an exit event
no containers to remove
0b20260e08c3a36efe1ee00288270ebe0e2253e4911e4e61dcfad783d5674b73
0d81528c765ecfb9768c2703be94d14a1efb724d2b4744b7e3ca02401d9ca751
1af85c61fd1492cda2c6f80f3b836ceaae8970903ce3397d94f495b1c76e9b33
1cca3b9066e97d49a5e36fc177dfde8be2e856d9c3f42a1ccb83e34186fc1e71
2ddbc6ddc393f6ad8ac73b75ed0e773692616f67412a15e9791bbabf905d3351
2efbfa26b503701f2a7bd81930e3c7d6e16f442fc987706b8cf7825267785575
02b0b6164dfdfb2117b90d6e3d974f81a0e571714d6889147d45b04b420c840c
3df698175aa979081884a4149bef899fd1b1cd99472f1a90bd9593a5d6fa47be
4dbf1cd466f5e1f86561b50731e2fd043b2884acded6835cf1e8249f2835feb4
4ee13b55a85c107d189708ff0fe54275095642163dc679856492f4d59d234d82
6ae9c3d1932311bfb5660b7879834794323882ed49e78763aae54a1cf0452b83
6bc5e69e068814af48f70e853cd1f4437bea0ceaa88252f0afb7b3a4ac1d06a2
6eef4ada64520ccf1aa78afcbc8ee6d9448d85eeefe2e090f79aefa8b6364ba4
9e7dfcdae569efd01596539eb24b5eb0e98ee6d9b3928d2b60bf9a5f178f763a
14d1903565eb87fe6d9c397e4df31ffddea088c51a3fa328da673f864671b466
14ed644423a6191a4bfac3adcaf6858df4c6198b57477b8ccfb6cf76924a27e6
17e496b21d5dcd9ccce129b9939b17f689d1465dce37dcc1807fa9343a05f05e
043cda1cb531af849c14248669aa1e793bd4045007c1141e8b4bf2bc007d6f11
48df1c4149c07ce41bf81e4ccc05ff6adec0ba93b6335a2d8c5ea5b9ac260558
51cb19158080633c46b8f914781918a79ae87d7e0042b985b0c59d4ff00ff4bf
57d3b89fb16513f4ff1a07fe3eee3bc636c0f41404d663a80c12a8dbe23084d8
66ab9edc6a8a7ef0ca76923a755116b5553777f9f2fac456b8364f300ed6aae6
78f24edd49b0373ed56477b3c6a11db7c9b44b8b6d45150fd929531c4a8737f2
235cce608a1cf294243698a2aab0d96ff835892f99744bac666460b92f12d4db
427ece00e517d300284ea64077b82b2fbd403c5ce8a878af94753370e2cbcb96
564b2c94ba4379f37d2dd54e6f9b2b6c21c78820c2993da1a8280ed4b27fac3f
702e88cc6ea59184292c8fe56720a651b47598367f439e3ec1bbfb4816ebfe19
959dd31a6d110d039a153318b5a14cf39a63097e63dfaa103a82a0f22c947b34
9509d71e311f273f84e43ece01c1c0c84a6bd9302d4ed5a63a41522b6db19602
27900aa4351fe529713cf39fffcee5b2d3e18dae6131337d7deab6cbee00169b
53032e088a74b6225e6844d3a4a0ff1aa7d6c18cdb26ec6e6356d21b430f0f07
77121a8ad958f0aa35011892e60f23146cdfc10cc63c9a7ca124b906273100c1
87875b0553c1d1dcd4fba5cd7184cc4ae724f1acb38afad5dc7d41f87cd72d53
66712142aeb20679a15850a03c68de9731636b26477bb70fa339d376ec995cf1
a2d8752cf9d7ec1c167ce4ec51b09f72c31e8df1198041bdb062601d6e0f878c
a89a1210d086948e74a8c25bb5ab11908311e9c9b0212a8fa27c794afc0de9f0
abbfc753bbc8802be48aae4b0cfecc93f7717a64623c8acbf0bc4e181b1a3362
af189e1cc6935c0d9c6e1a87098eb155a36002df296871a2115e43aff387e00e
af604af0f3181c242f5413338faa445a4ea8c7b68e9dea1b320294d52050f19c
b38a6c1c216b80234fc23eb6f3088936609d598d3b02a02bb6e6ec7e313bf076
b497c4d5aae81293d1d9f3de038f9f5e9424bbddec309453acdce59c34116439
bba5b53debea39e28da86151092e0e4bc06e3d190ee0f12d1dfa818079020a98
c06c4e4f9c86f55c6b25773483ff98b4c52dcdd0b22691ea2a87da480b8a4f76
c3806d2697fe0ccebeb3b9753a99de89274358d31c2e55d7a47ff7ac712183bb
c31410d0812b6f7fdafd76f6690d38519c6b9ce94d04082311c2dfe587d6f7eb
d9aaef8e0a9dd217a528878f73557617cd3b04d606c097c59be2b740a0c66e8a
d9486e023b73e329670103803df496224999d64b972cba0545e4b0999b280fc8
db9a2031b6a0ac03d629e3d65c18144c83ffda44f82d1c4e3890c872ca4d375e
e3dd70e132d794c930ce2b35654a10e03f03bb8be90b0f93fac3c6e2b73168b1
e59ead999363d7b047aa78caa39f4d24c958b65b02fcdef904c2a5817c2470b3
e506e032d2889b36f3f214fce8f0fbdfcbb1239f349f5e8efc0452bfba1e6ec2
e6919b37e3fa184007db03146b7204588f77b494ff9f6cf2eab6eb6f4a4fdedb
ecb9bd0ed296154ede9ca793c96f1813d1acc7d016abdd7a2383569a2b0ebee7
f1c2f9b52831c8f9c5a98eb094e8c9459b3724581bda3b35046d84e7141be42d
f6c66ec154d5213e63bd34c51362685be0faaf2b9d29fef8d3f0503a0727ba18
f7bccfeaf334ef783aff9964b03fa74d1a87ddc8f9065174dc0fd130230c81a3
f17b10b599b7179ff982a03ff57e29d284ee7cd539837c7301d718bd3b0fb28d
f45e17085dc02d35078ca30b13c1723ea4f294ef4c8e1661d411a5f5fda14948
fbe4c91776aef9550d6307628cf12f4d0bc5c2e1aa89791806d80cd3d0ad1a6e
fda530926f3ff29d3a67247dd591817d347199a919ca792fd48a8b303a66ab2c
fedad1199de437602dd87b2628936d2c60b68d6ba3422f05fff0d8d84c72393e
ffbaff6b75f04f4500a2262f93a57f022d9486be459843f8d2f5f7500450f8d4
fidesops_app-db-data
Error response from daemon: remove 7ec33418eead49fc08d5a15f44e124b05aca8afdfae9c5d84aa60e6be002a9e6: volume is in use - [f0972005382a684aa39a62e179a21b34c01223b1798d992d60513e8eaec923e7]
Error response from daemon: remove bc5a7b76ea038b830716dc5c555dda04f9b303603da624b1f07208eba1a1f19a: volume is in use - [f0972005382a684aa39a62e179a21b34c01223b1798d992d60513e8eaec923e7]
Error response from daemon: remove d5536e107b66b32df27868dcb4bc5698022569974ec4ffc35c88aeabb5be9771: volume is in use - [f0972005382a684aa39a62e179a21b34c01223b1798d992d60513e8eaec923e7]
Error response from daemon: remove ddde69b9fef32806244ec2d9c43bccdd0cf8698e688e6da37160734ebcc9203c: volume is in use - [9726f1077a7396f547e7c219c0229a2a7a9d4a308359a745a64eff1dd6c8a627]
Error response from daemon: remove f5fd255f40b7812aa464b5507c1ac0017696be9555754dcbcecefa6b551b2323: volume is in use - [a02f61cd12103fb5d1a4c015c5bf7e35074d74f1a5eeca54e0cdca741b8565b1]
no volumes to remove
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all volumes not used by at least one container
  - all images without at least one container associated to them
  - all build cache

Are you sure you want to continue? [y/N] y
Deleted Containers:
9726f1077a7396f547e7c219c0229a2a7a9d4a308359a745a64eff1dd6c8a627
4dd064c3224874d728ab621a3b82970f6cfe35303cbe68ba60c063b68cfd2fff
a02f61cd12103fb5d1a4c015c5bf7e35074d74f1a5eeca54e0cdca741b8565b1
f0972005382a684aa39a62e179a21b34c01223b1798d992d60513e8eaec923e7

Deleted Networks:
fidesops_default

Deleted Volumes:
f5fd255f40b7812aa464b5507c1ac0017696be9555754dcbcecefa6b551b2323
bc5a7b76ea038b830716dc5c555dda04f9b303603da624b1f07208eba1a1f19a
d5536e107b66b32df27868dcb4bc5698022569974ec4ffc35c88aeabb5be9771
ddde69b9fef32806244ec2d9c43bccdd0cf8698e688e6da37160734ebcc9203c
7ec33418eead49fc08d5a15f44e124b05aca8afdfae9c5d84aa60e6be002a9e6

Deleted Images:
untagged: fidesops_docs:latest
deleted: sha256:ae9202df4a544949bd64e2e5d40bb8938002171b5d5a1ca59c92f48d6356267b
untagged: postgres:12
untagged: postgres@sha256:e6ffad42c91a4d5a29257a27ac4e160c3ae7196696b37bf2e80410024ed95951
deleted: sha256:97bdb4126e9947ee7b83b0fb522dd45a73256c6ee3918b7311311d9f286bce49
deleted: sha256:c1bc07867d28cbd57c3ddc8fc8741c8d58df59d353950e0f1484a034f230c459
deleted: sha256:9ed382948a46b5b8ffde98e6cf017f4af146698aaa6279608aab677af4724622
deleted: sha256:61f72b91d4a3c49d52e82b1ee0db3f7bfcd470c6f9c930eac385963ce72c43f9
deleted: sha256:d7e2cc184868da432c25279ab919ffa379a89e243f0a66d4b18f26c7f7a3c872
deleted: sha256:6511bfc891bea0dfb46070f43e3ee32209f0c3570734e81df255804035dd1e67
deleted: sha256:108a44476a70b4aaee7cb8e5f9a63ef4e5ea33fe767d242ea1f4ca9fc22894d6
deleted: sha256:5008200b0021c87da500d8d2a65851763db3b7233504538cb520bcaf3a341cbf
deleted: sha256:ae31244a840f2fda29f0543e53cc76b5861b56cb617edb79a6f9ade610a11c3f
deleted: sha256:6601e14a3e0a99b256260b5f92f674bd6e617762d095cd7d5e2151cc5af2ca61
deleted: sha256:4cab956a8e31306beabb9ef4042d73c833704612b6af3be2edc81e6aed5079c8
deleted: sha256:017ce9e16a168f2808e7a1a9f89912432f96e8c5958136c45f15791045f38ec8
deleted: sha256:0aa9d420eb9d49f56624f0d3b30d2fff7c98bc7540444568104df6b1f043a235
deleted: sha256:5978b6b69f17c6a0254d34a4180f1f2c8d42f54e6ffe96303bacd2db8e30955d
untagged: mariadb:latest
untagged: mariadb@sha256:88fcb7d92c7f61cd885c4d309c98461f3607aa6dbd57a2474be86e1956b36d13
deleted: sha256:ea81af801379dae14e596b55612f355a63cfca2ed53cf27d047340f4890cf25c
deleted: sha256:e9312d79c459a24cc05f5fc5ab56c9377eaad488658201f5b0fe201c2a48295c
deleted: sha256:c2b4badaebf53753fda1bc03a9fe1ad7775743a1cc323e83d34563dbd8ff6bb9
deleted: sha256:10742ba639c781b839b1083c9fd61dfdddbd5976f618e020aafbd225009aedc1
deleted: sha256:52ee5d95c9a7244ffb8778f23f5859dd342be8012504b3dcdb4b26342bb93af7
deleted: sha256:f1657f2ba3ba44c0a48290d4946fdd496445a283efa359295ce320c4a6dedc96
deleted: sha256:dda7586ce4c6f476a74381999b29ebb2b2f26d4763a86d8fcc786965ac9a4a46
deleted: sha256:d8cf5c155c3717258bb56ffc308f63ff46956931aedcda14a4c43d7ccf3bfc99
deleted: sha256:17f2d9abcdedc83b91f7e5775d0525b44cc56b47362bda3015d8b2d2326ab19b
deleted: sha256:62286867b317a35ff1b2de0b88f5d93ea2363ee79d7afcd1e7c9d021ecd89e7c
deleted: sha256:8475e5110053484d93082b6e7243196fa32d05eab8501140dc1ba5f58118fe15
deleted: sha256:a790f937a6aea2600982de54a5fb995c681dd74f26968d6b74286e06839e4fb3
untagged: mongo:5.0.3
untagged: mongo@sha256:4088649f737cf704deaf350ccd5ad8045552c5a0f8a5a2e81c1c23e280db2d80
deleted: sha256:addfd455919fa552fae7dc3cfd6d8fa690c784f7bf07aeaa866e5c87ab106063
deleted: sha256:7df053c031c7d0727a05828e1546c062387476b42fdca5bf2b4fcf9ec4b0e204
deleted: sha256:6c813c27571f3039037d49b5d4134c8966800acc548bafb641f83b61768d8e9d
deleted: sha256:49656de205879d210de220c06b13f71ddd0cf19ee583e90e282cd2815475d127
deleted: sha256:cfb508435f80c672e97c434fd613b562eb13023e8480e761898578dae008de8c
deleted: sha256:1bb5b57dd0b760775fc04947fb696d2828012f431cb718dd531f71096e3f680c
deleted: sha256:cacaa1471b51424c9d0558a727682c52baad26ab5360bd228d25ed5a232837d3
deleted: sha256:1f2eaff47426c640b489a65b53f5f938fcc6f705a94343f5ad4cbc93a3921789
deleted: sha256:69abb39306049f0b856e48ef67a45fe5c9bfbe95b665a84b8d8c897a278fdc38
deleted: sha256:28d8b9be682314cea28896d582b72bd43c4ba03454ae68778a53471151a9e354
deleted: sha256:350f36b271dee3d47478fbcd72b98fed5bbcc369632f2d115c3cb62d784edaec
untagged: fidesops_celery:latest
untagged: fidesops_fidesops:latest
deleted: sha256:9fa5d232750a4065bda53e8210ec144132dc89d8bf38a5401803040378a106e4
untagged: mcr.microsoft.com/azure-sql-edge:latest
untagged: mcr.microsoft.com/azure-sql-edge@sha256:dfce05471e259d330e00bc9e66904254d75b714b35edc81495b4faee4a2dba96
deleted: sha256:c0db2d0e0637471442f5adb4e4aa69b42030e5995558c8333fca17a77f5fd2ad
deleted: sha256:ca9c7a699c18d824a10606edb95121ccd6620935e982d6a6586946e4dba638c0
deleted: sha256:aee8529969a0abb060d4094439026969fd78a4ca4a6a91b13be3dc4cd6f047c1
deleted: sha256:0a84435aa151e0762a785bc5555804ef25e5f2467719834636b4d5bac46c964b
deleted: sha256:8887fedcfe4ad5bbb757a17d445fc389185e9257d407094629882f0abf0dbbfe
deleted: sha256:39187d3cc25efbcff7f9a285756f886cbc64aa3683c53a0251e01cc309dd9ae7
deleted: sha256:152e73f35c5e6d887b1387eedeb95fb6bcc2c5d8386f751515e88edd7e2bbc86
deleted: sha256:965bbaa6e60f20d3568458f7b1c8caa9d47bb8e9f6f7b93a3014660fd48776a4
deleted: sha256:05e000c98c3f730ace050e5ef35dc3d9b1437e5785e6b665cca20e753bfb135d
deleted: sha256:b885a54c12dafece0a9c5a051e5ea9c7066b385c76ae16ed05c99deb51d1f00f
deleted: sha256:093ca61ef3f098a0eee0b4d5eb1309baaaf6e951ebb6a3d2be6377753b17277b
deleted: sha256:abf82ff382a958fa1beb54b7c2cf45e1e5c80fe82f65ace56b7fb7cec47c4400
untagged: mysql:latest
untagged: mysql@sha256:a840244706a5fdc3c704b15a3700bfda39fdc069262d7753fa09de2d9faf5f83
deleted: sha256:0ef9083d9892db139e2b7c81fd133836e7aa32db2b5f1d5702e2593100fa1417
deleted: sha256:d4e6990adce2efbe5528baabe5786cd87a18c7b8628818eef0c7407b2feebc6a
deleted: sha256:84714532693fd93dc053a83f131e20abafb290b2593ab0425ba34f45fe2b032e
deleted: sha256:e491806b556593e8ab0ed17a1548c5016047182c3b00e3458bbc65c7b9f00bdb
deleted: sha256:72cbf6dbff0b8a89e0a24e5f7077635242be206b73606852a1c885da8330b112
deleted: sha256:ac73cdbbaa26724241c9f71ccc017ebfe189509b15c7ec79cd211dffbed0fd65
deleted: sha256:d535b9afb1924ccd3dc0b9a665e51a0d1d9ccbe7b4894f3d8c0d3b1aa8a1cc66
deleted: sha256:1caee35c1ad3324aea982fc3742aa2b067384bf4a6d6023e8a3fe688fb56773f
deleted: sha256:6711a10388f48ae0bfed91f7a72a24944d5c8c76be2d69667fd9d22848a417e6
deleted: sha256:213f68209a99da399858ec3ca9a4e3d331b7cded5c43bbc27e3adaa916caa9c8
deleted: sha256:893dd0d26de3f7c4ca8a19ccf3dd75b27f259d105e059c37ad1eaadbbe21f52f
deleted: sha256:6890009cd8a84c97eb3ccda36e7744db60b83299ee55d9afd0503bbe7158d61d
untagged: redis:6.2.5-alpine
untagged: redis@sha256:fa785f9bd167b94a6b30210ae32422469f4b0f805f4df12733c2f177f500d1ba
deleted: sha256:2d113b08aca489af751537d7fd54064d4c8e10bf4635b608602ad9c7ecec0dc7
deleted: sha256:1d3403c5cf2a3b5565ca0f4e59092b013868420e38f5aa5296b2a39b5a6ee79c
deleted: sha256:815c1a1618af9eaef9d37a038be3bc1668ae07a21ffa3beb6eddbd1d09b67fb0
deleted: sha256:69d36589c1ea092206b0831267d912098c4ec92560b296db79105ca9b44c90fd
deleted: sha256:20fe8a9569ceab4a437d50c2dcec16eae7180990b0eda8d0b9c5f1e02278f320
deleted: sha256:1c9cdca6532f6bb859175ab26acbffd44bcf30ae1319105e597afcff54209804
deleted: sha256:ee420dfed78aeb92dbd78bf0e97e3a3cad5ccfbb59fa5dac4e04639210cc7905

Deleted build cache objects:
cne65ibx4dvor2p7cpy87xw2b
nokhjofnv3msm1r26hvfx0qu8
jlybyzw1r2mhwcb5y92ac5qov
8o9qj1pd7qig19gbh68ypzka1
udeq6t9beg9xyahyamv9a8xqm
12fdjhy6m2skc6u82j3vffxd7
2n9k19z8e3cz1201x29larizc
l4k8tk37jjau52y4yrsjlohqo
9c0fc1hkhd307zvsiciwp2coy
knbw16iwckyckznhi73t806zz
rxn8a7gap4qukrftb3jnnd5ku
bgcka4jqwk78mo1yq8zhuyvqo
knqjob2puaf8v1v6eilobfd8p
l158r40tszx19pejw8z994qbi
pzc8fbprlfmyaq9yq7q6pukbu
kpnn8ex6p45v4v9ra2jnn62oy
kpckvjjjnadleeek6sim8d9x6
tkgm3jzm0i70dfoe1z1rmbqnc
c4rck7vokiwqxoots05olaljw
rwu3qssgchydlkbo9ut63lzbc
uh9mjrpy481mdvb11c2j03hen
c4zcki1j20w6pzazkoxlrghos
iz6zgxo1vufvjva0q0sdqxz3y
sxotopn642sc079rsj80v2ibg
tlhoqr881uft899cyep3yqpxw
vh4onprc0sk7677c3d7vp20vx
rafalganfx2ipw05jp4ioxih4
bmaa2thx6d4mwwh3wwuqjflu9
thhirik9g2xnhmq43nxg0fh6m
tlljew2jnssvmte6xcz2aj0bx
zutct9plqwwcy4f7d08j7osb4
zc1riizoab3lue6v7k4hu3xco
v65f93x2znzfu8pqbrhk8qye3
dcthm1tux7dbikv86f7t8zxeh
4dohpm9fsmpfml366gv0qhuvz
71ffiqlj1iinhrsmsndcdz2ln
89liknw10inp1xsf6esogpcnl
i7108xgvty5z8oc78kmi53fst
y57h519j4p8u4vv7vibpebxk2
4dzlrx2is8gtdctjif2kxfvwn
k2ltdt4ru1c14r5v9hmzhnj8d
iflobkbcxqf1aeq3qdzirk843
dptj3ru69x2dr4tn895o4ddqd
k3kaej0oyh0zfzpvm1exqr55u
owct3yosfmjwe0l32h5l7gi43
usvlebmm3goi2pxn5u2dbh3qh
ng7ry11xsz8pdht1cfqt6ue3n
kj9uwsnyhfxbcyz6m81ig7knx
4abxdklxxpkxu0inju2ct2auv
0bouvgka0umawey8i0vyh0c1a

Total reclaimed space: 5.93GB
Clean complete!
```